rocky/rocky/views/mixins.py

```diff
@@ -46,6 +48,19 @@ class OriginData(BaseModel):
     boefje: Boefje | None = None
     params: dict[str, str] | None = None
 
+    @property
```

Comment:
Another solution, assuming a property would be convenient to use within the templates, is to create a `is_older_than` method, and still have this property (without arguments) calling the `is_older_than` method with the default value.

But since I haven't seen any uses of `max_age`, I suggest leaving the `max_age` value out and just have
```
@property
def is_old() -> bool:
    ... # check against `ORIGIN_MAX_AGE`
```

Reply:
Creating a property (for easy access with a default) makes sense, and creating a is_older_than with an argument makes sense.
Especially when we can replace the default based on the freshness expectations we have of each OOI with a data-driven timeout later on.